scripts/analyst_collective/runner.py

```diff
@@ -0,0 +1,64 @@
+
+import sqlparse, psycopg2, sys, os
+
+class Runner(object):
+    def __init__(self, config, creds, models_dir):
+        self.config = config
+        self.creds = creds
+        self.models_dir = models_dir
+
+        self.connection = psycopg2.connect(creds.conn_string)
+
+    def models(self):
+        return self.config['models']
+
+    def drop_schema(self):
+        sql = self.interpolate("drop schema if exists {schema} cascade")
+        self.execute(sql)
+
+    def create_schema(self):
+        sql = self.interpolate("create schema {schema}")
+        self.execute(sql)
+
+    def clean_schema(self):
+        self.drop_schema()
+        self.create_schema()
+
+    def execute(self, sql):
+        debug = sql.replace("\n", " ").strip()[0:200]
+        print "Running: {}".format(debug)
+        with self.connection as connection:
+            with connection.cursor() as cursor:
+                cursor.execute(sql)
+                print "  {}".format(cursor.statusmessage)
+
+    def interpolate(self, sql, model_name=""):
+        try:
+            return sql.format(model=model_name, **self.config)
+        except KeyError as e:
+            print "Error interpolating key: {{{error_key}}} in model: {model}".format(error_key=str(e).replace("'", ""), model=model_name)
+            return None
+
+    def add_prefix(self, uninterpolated_sql, model):
```

Comment:
Agreed - as long as we're marking up which interfaces a table/view implements (in comments or in a separate table), then we can allow the table name to be variable as well.  This creates a bunch of new complexity, though. I think we can leave this as-is for now.